src/Module.Server/MapRotation/e_maps.txt

```diff
@@ -1,2 +1,4 @@
 crpg_dtv_weepinghill
 crpg_dtv_rosethornecastle
+crpg_dtv_encampment
+crpg_dtv_qalat
```

Comment:
Qalat and Encampment are the default dtv maps. They are used to bootstrap the server as the server requires at least two maps to load. So no matter what maps are in e_maps , Qalat and Encampment are always part of the rotation.